README.md

```diff
@@ -295,19 +295,21 @@ HTTP/1.1 204 No Content
 Request:
 
 ```sh
-curl --include --request POST http://localhost:3000/sign-up \
+curl http://localhost:4741/sign-up \
+  --include \
+  --request POST \
   --header "Content-Type: application/json" \
   --data '{
     "credentials": {
-      "email": "an@example.email",
-      "password": "an example password",
-      "password_confirmation": "an example password"
+      "email": "'"${EMAIL}"'",
```

Comment:
These are not copies of the scripts. They are separate from the scripts and are an attempt at describing a valid request.